pipeline/released.groovy

```diff
@@ -0,0 +1,111 @@
+// Script to update the container build information into QE recipes.
```

Comment:
Could we fix the documentation? I believe the purpose of this script is to trigger when a RH Ceph is released and execute Tier-0 using the bits available in the external repository.